src/orchestrationclientinputdata.ts

```diff
@@ -2,11 +2,25 @@ import { HttpCreationPayload, HttpManagementPayload } from "./classes";
 
 /** @hidden */
 export class OrchestrationClientInputData {
+    public static isOrchestrationClientInputData(obj: unknown): boolean {
+        const typedInstance = obj as { [index: string]: unknown };
+        if (typedInstance) {
+            // Only check for required fields.
+            if (typedInstance.taskHubName !== undefined
+                && typedInstance.creationUrls !== undefined
+                && typedInstance.managementUrls !== undefined) {
+                return true;
+            }
+            return false;
+        }
+        return true;
```

Comment:
So we return `true` if it's *not* a typedInstance?

Reply:
woops! Will fix that :)